advanced/am-pattern-2/README.md

```diff
@@ -0,0 +1,203 @@
+# Helm Chart for deployment of WSO2 API Manager Pattern 2: Deployment with a separate Gateway and separate Key Manager
```

Comment:
@Chathuru please include details about WSO2 product Docker image types in the README documentation. Please restructure the README as per [1]. Feel free to drop in your suggestions and concerns.

[1]: [IAM pattern 1 README documentation](https://github.com/wso2/kubernetes-is/blob/v5.10.0.1/advanced/is-pattern-1/README.md)